packages/frontend/src/hooks/useActivities.ts

```diff
@@ -3,6 +3,9 @@ import { useEffect, useState } from "react";
 import { metromApiClient } from "../commons";
 import { useAccount, useChainId } from "wagmi";
 
+const TIME_RANGE = 60 * 60 * 60 * 24 * 7; // 1 week
```

Comment:
This is incorrect, we have to remove a `60`.

Reply:
Addressed in 44b2a2e949ea99ea6ce5ccf544682a243c1fbcc3